src/if_perl.xs

```diff
@@ -40,7 +40,7 @@
 /* Work around for perl-5.18.
  * Don't include "perl\lib\CORE\inline.h" for now,
  * include it after Perl_sv_free2 is defined. */
-#if (PERL_REVISION == 5) && (PERL_VERSION >= 18)
+#ifdef DYNAMIC_PERL
```

Comment:
`PERL_REVISION` and `PERL_VERSION` cannot be used here, because they are defined later.